rust/protocol/src/sealed_sender.rs

```diff
@@ -704,6 +798,55 @@ pub async fn sealed_sender_encrypt<R: Rng + CryptoRng>(
     sealed_sender_encrypt_from_usmc(destination, &usmc, identity_store, ctx, rng).await
 }
 
+/// This method implements the single-key single-recipient [KEM] described in [this Signal blog
+/// post], a.k.a Sealed Sender v1.
+///
+/// [KEM]: https://en.wikipedia.org/wiki/Key_encapsulation
+/// [this Signal blog post]: https://signal.org/blog/sealed-sender/
+///
+/// # Contrast with Sealed Sender v2
+/// The *single-recipient* KEM scheme implemented by this method partially derives the encryption
+/// key from the recipient's identity key, which would then require re-encrypting the same message
+/// multiple times to send to multiple recipients. In contrast,
+/// [Sealed Sender v2][sealed_sender_multi_recipient_encrypt] uses a *multi-recipient* KEM scheme
```

Comment:
I believe this link should be using parens, not brackets, since `sealed_sender_multi_recipient_encrypt` is a destination. (Maybe it's a rustdoc extension? But still.)

Reply:
Ah yes! It is a rustdoc extension, but it appears to work great with parens!